### ISSUE TITLE
Deprecate X/Twitter

### ISSUE THREAD

**Issue by austinlparker** [open-telemetry/community#2444]
See also https://github.com/cncf/foundation/issues/893

Given the recent surge in activity by the observability and cloud-native community towards ATProto/Bluesky, I suggest we move to deprecate our Twitter presence and use Bluesky/Mastodon/LinkedIn as our social channels.

**Comment by jpkrohling** [comment]
As much as I don't like its leadership, I don't think we _need_ to deprecate X: we have over 17k followers there. Besides, we don't need to deprecate X in favor of something else: we can increase our presence on Bluesky while keeping our X presence.

This comment from Chris A. is what I think should be our stance here as well: 

> We won't deprecate a particular social channel but plan to support all channels are communities and contributors partake in.

**Comment by woody1872** [comment]
TL;DR - I agree with Juraci. No need to deprecate Twitter, just start a Bluesky account and build up a community there too if there is demand for it. 

I find that CNCF thread extremely embarrassing to read. Like Bluesky, it's an echo chamber and it's full of politically-motivated nonsense that has absolutely no place whatsoever in an open-source project. Mass downvoting posts you don't like for purely political reasons to stifle debate and opposing viewpoints to further a political agenda. I absolutely detest that kind of behaviour and it's directly responsible for much of the violence we see today. At some point people need to get a grip and accept that healthy debate and differing ideas in all aspects of life are actually very good things. Makes us human.

On a technical level, I disagree with much of what's been said about Twitter - which I refuse to call X btw (sorry Elon). For example, I don't agree that it's more closed platform at all. In fact I see it as significantly more open than ever before, welcoming people who have been silenced previously and giving them a voice. Community notes is a phenomenal tool for fact-checking as well, it works incredibly well. It certainly has many problems though of course (bots for example), but what social media doesn't? Instagram has been directly linked to the suicide of kids, TikTok is utter brain-rot, Facebook is basically a data-mining tool, YouTube has mountains of scam ads, etc.

The trouble is today too many people want these echo chambers, so nobody can challenge their world views. A sad pathetic reality we are currently living in.

Anyway, it would be absolutely amazing if we could please avoid the CNCF / OpenTelemetry becoming a political party and focus on the task at hand? Vendor NEUTRAL telemetry data collection. Keep your politics to yourself and never let it infect the projects goals, CoC, T&Cs etc.

I don't really want to comment further because it just results in hatred and discrimination against people like me who hold opposing views. I'd rather just see the project go from strength to strength with good people.

Have a great Christmas and New Year.

**Comment by danielgblanco** [comment]
I'd generally agree with growing our audience on Bluesky before shutting down X, however this issue can help lay the path forward. We currently have 1.5K followers on Bluesky, that are probably more engaged than those on X. I say "probably" because I don't have an X account, which means I can't read any of the content published there. This also relates to

>I see it as significantly more open than ever before

Without having read the full issue, I would agree with the fact that X has become more of a closed platform. And this is something important, as we cannot expect every user that wants to read about OpenTelemetry on social media to have an account on a given platform. One of the advantages of Bluesky, or even LinkedIn, is that posts can be shared and read by anyone without having an account.

As I said previously, I don't think there's pressure in closing our X account as we're not under the same circumstances of public opinion and disinformation that made other accounts like NPR, The Guardian or more recently major UK universities leave the platform. However, in the interest of information sharing, I'd support the decision to stop focusing on X in the future and, if we see low levels of engagement, deprecate it.

**Comment by mx-psi** [comment]
I think it is clear by now that X is not aligned with the community values and that it is, to say the least, a challenging environment in which to ensure our values and Code of Conduct are honored. I support leaving X and re-enforcing our presence on other social networks.

**Comment by austinlparker** [comment]
This topic will be added to the GC agenda this week (1/23) -- if people have any further thoughts, please get them in before then.

**Comment by austinlparker** [comment]
The GC has voted on and accepted this proposal.

**Comment by austinlparker** [comment]
Website, GitHub profile have both been updated. Twitter has been removed from our Buffer channels and the profile/pinned tweet have been updated to redirect users to our other channels. If anyone finds other links that need to be updated, please handle them on a case-by-case basis.
